### PR TITLE
Animation contributions from Tenshi

### PR DESCRIPTION
**Pull request by zontreck**
```
[10:09:10] Serenity (tashia.redrose): I can open the pull request against your fork of OpenCollar. If i do so- do you consent to us reviewing and pulling in for consideration your changes or contributions?
[10:09:28] Serenity (tashia.redrose): keep in mind i can only open a PR if the repo is a direct fork of OC
[10:10:06] Serenity (tashia.redrose): i mention this because you stated you put them in a repo named tenshilove
[10:10:18] Tenshi Love (tenshi.zeffirelli): yes of course i consent :)
```

### CODE REVIEW

**Comment by zontreck** [comment]
Partially addresses #370, #203

File: res/anims/poses/alternate_oc_anims/naduwide-arc.bvh [hunk 1]
```diff
@@ -0,0 +1,304 @@
+HIERARCHY
```
Comment (zontreck) [inline res/anims/poses/alternate_oc_anims/naduwide-arc.bvh:1]: This animation had one of the arms cross inside the body. Specifically the left arm went inside the chest while testing

File: res/anims/poses/alternate_oc_anims/doggie.bvh [hunk 2]
```diff
@@ -0,0 +1,122 @@
+HIERARCHY
```
[the remark this answers is not in the document]
Comment (tenshilove) [reply res/anims/poses/alternate_oc_anims/doggie.bvh:1]: it's meant to be a more static thing, an alternative more than a replacement. I'll try to add it, when I get more experience animating. These are essentially first tries.

File: res/anims/poses/alternate_oc_anims/naduwide-arc.bvh (hunk 1 above)
Comment (tenshilove) [reply res/anims/poses/alternate_oc_anims/naduwide-arc.bvh:1]: i know. Whenever I tried to fix, it got boked so i left it as is.

**Review by tenshilove — APPROVED** [review]

**Review by tenshilove — APPROVED** [review]

File: res/anims/poses/alternate_oc_anims/doggie.bvh (hunk 2 above)
[the remark this answers is not in the document]
Comment (zontreck) [reply res/anims/poses/alternate_oc_anims/doggie.bvh:1]: I am not sure i can approve this animation as is, the other animations are fine, its the two i marked that have issues and i'll remove after merging

**Comment by zontreck** [comment]
TODO: Remove post-merge the doggie and naduwide-arc files

**Comment by tenshilove** [comment]
I'm glad my animations helped :)

**Pull request merged**